adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -127,6 +129,33 @@ def connect_normal(self):
                     self.reset()
                 continue
 
+    def create_ap(self):
```

Comment:
I like this better: `create_AP(self, ssid, password)`. Dont need channel/timeout since they're already set in the esp32spi class.

Reply:
The reason I made it like this was to match the current pattern of the WifiManager, which requires a positional arg `secrets` object that contains the SSID and (now an optional) password. 

I figure since you can't currently be in both AP mode and Client mode at the same time, that it would make sense to re purpose the secrets object instance variable as a place to configure your AP SSID and optional password. 

Another reason (though I don't think this is a big deal in either case), is our example files wouldn't have to provide a default dummy SSID and password.

I don't disagree though, I just think it makes sense to be consistent.